### ISSUE TITLE
The build breaks in Rust 1.38.0

### ISSUE THREAD

**Issue by frosklis** [dfhoughton/pidgin#20]
I had pidgin as a dependendy of a dependency (two-timer). When it was upgraded from 0.4.0 to 0.4.1 [my build broke for Rust 1.38.0](https://github.com/frosklis/dinero-rs/runs/2214508177?check_suite_focus=true). Interestingly not for stable.

Here's the log:

```
   Compiling pidgin v0.4.1
error[E0596]: cannot borrow `phrases` as mutable, as it is not declared as mutable
   --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/pidgin-0.4.1/src/pidgin.rs:626:27
    |
601 |     fn find_character_classes(&self, phrases: Vec<Vec<Expression>>) -> Vec<Vec<Expression>> {
    |                                      ------- help: consider changing this to be mutable: `mut phrases`
...
626 |             v.append(&mut phrases[char_count..l].to_vec());
    |                           ^^^^^^^ cannot borrow as mutable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0596`.
error: Could not compile `pidgin`.
warning: build failed, waiting for other jobs to finish...
error: build failed
Error: Process completed with exit code 101.
```

**Comment by dfhoughton** [comment]
Ah, sorry about that. I'll fix it. I just removed that `mut`, thinking it was unnecessary. The tests all passed. I guess I need to add a test.

**Comment by dfhoughton** [comment]
```
   Compiling pidgin v0.4.2 (/Users/houghton/git_projects/pidgin/target/package/pidgin-0.4.2)
warning: variable does not need to be mutable
   --> src/pidgin.rs:601:38
    |
601 |     fn find_character_classes(&self, mut phrases: Vec<Vec<Expression>>) -> Vec<Vec<Expression>> {
    |                                      ----^^^^^^^
    |                                      |
    |                                      help: remove this `mut`
    |
    = note: `#[warn(unused_mut)]` on by default

warning: 1 warning emitted

    Finished dev [unoptimized + debuginfo] target(s) in 38.57s
   Uploading pidgin v0.4.2 (/Users/houghton/git_projects/pidgin)
```
Heh.

**Comment by dfhoughton** [comment]
@frosklis , tell me that fixes it for you. I published 0.4.2 just now. Two-timer compiled fine for me in both stable and nightly.

**Comment by frosklis** [comment]
> @frosklis , tell me that fixes it for you. I published 0.4.2 just now. Two-timer compiled fine for me in both stable and nightly.

[It did](https://github.com/frosklis/dinero-rs/runs/2228356013?check_suite_focus=true) Thanks!
